src/scenes/NurseryRouter/UndoWithdrawalModal.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import DialogBox from 'src/components/common/DialogBox/DialogBox';
+import Button from 'src/components/common/button/Button';
+import { NurseryWithdrawalService } from 'src/services';
+import strings from 'src/strings';
+import useSnackbar from 'src/utils/useSnackbar';
+
+export interface UndoWithdrawalModalProps {
+  onClose: () => void;
+  row: any;
+  reload?: () => void;
+}
+
+export default function UndoWithdrawalModal(props: UndoWithdrawalModalProps): JSX.Element {
+  const { onClose, row, reload } = props;
+  const snackbar = useSnackbar();
+
+  const onSubmit = async () => {
+    const response = await NurseryWithdrawalService.undoNurseryWithdrawal(row.id);
```

Comment:
Is this the only place we'll ever use this? I think it would be more consistent to put in redux using the async thunk flow, but if this is the only place then I can be okay calling it here.

Reply:
Yes, it's the only place I guess